### ISSUE TITLE
[Mailer] External CSS files in Twig does not seem to work

### ISSUE THREAD

**Issue by weaverryan** [symfony/symfony#31746]
**Symfony version(s) affected**: 4.3.0

**Description**  
In the Mime Type docs and in Fab's original presentations, it was advertised that, when inlining CSS, you could point to external files:

```twig
{% filter inline_css('@css/foo.css') %}
...
{% endfilter %}
```

This doesn't seem to work. The Twig filter - `CssInlinerExtension` accepts the `$css` passed to and passes it directly to to the TijsVerkoyen lib:

https://github.com/twigphp/cssinliner-extension/blob/295cc11b8f1fe31ff95475ecbbc8c0d966f4f647/src/CssInlinerExtension.php#L35

In other words, the path to the `@css/foo.css` file is passed, but its contents are never read - it's just seen as an invalid CSS string and discarded.

**How to reproduce**  

**Possible Solution**  
I'm not sure... another problem is that the `@css/foo.css` is never resolved using the Twig namespaces. That needs to happen first before anything can read its contents.

**Comment by weaverryan** [comment]
Happy to report that I'm wrong again. The missing piece is `source()`, of course :) 

```twig
{% filter inline_css(source('@css/foo.css')) %}
...
{% endfilter %}
```
